packages/augur-ui/src/modules/account/components/funds.tsx

```diff
@@ -15,26 +15,25 @@ import {
 } from "modules/auth/helpers/login-account";
 
 const Funds = () => {
-  const { loginAccount, actions: { setModal } } = useAppStatusStore();
+  const { loginAccount, theme, actions: { setModal } } = useAppStatusStore();
   const {
     totalFrozenFunds,
     totalAvailableTradingBalance,
     totalAccountValue,
   } = getAccountFunds(loginAccount);
-  const { tradingPositionsTotal } = loginAccount;
   return (
     <section className={Styles.Funds}>
       <h4>{TOTAL_ACCOUNT_VALUE_IN_DAI}</h4>
       <div>{formatDai(totalAccountValue, { removeComma: true }).full}</div>
       <LinearPropertyLabel
         value={formatDai(totalAvailableTradingBalance, { removeComma: true }).full}
-        label={AVAILABLE_TRADING_BALANCE}
+        label={theme === THEMES.TRADING ? AVAILABLE_TRADING_BALANCE : AVAILABLE_BETTING_BALANCE}
```

Comment:
I'd use a variable for this since we repeat it so often, before returning make a `const isTrading = theme === THEMES.TRADING;` so we can shorten up some of these ternary operators